actions/databases.go

```diff
@@ -0,0 +1,290 @@
+package actions
+
+import (
+	"errors"
+	"log"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/gobuffalo/buffalo"
+)
+
+// DatabasesList gets a list of databases for a given account
+//   If the `all=true` parameter is passed it will return a list of clusters in addition to instances.
+func DatabasesList(c buffalo.Context) error {
+	// if all param is given, we'll return information about both instances and clusters
+	// otherwise, only database instances will be returned
+	all := false
+	if b, err := strconv.ParseBool(c.Param("all")); err == nil {
+		all = b
+	}
+
+	rdsClient := RDS[c.Param("account")]
+	var clustersOutput *rds.DescribeDBClustersOutput
+	var instancesOutput *rds.DescribeDBInstancesOutput
+	var err error
+
+	if all {
+		clustersOutput, err = rdsClient.Service.DescribeDBClusters(&rds.DescribeDBClustersInput{})
+		if err != nil {
+			log.Println(err.Error())
+		}
```

Comment:
i think you can one-liner this since both vars are defined

```
if clustersOutput, err = rdsClient.Service.DescribeDBClusters(&rds.DescribeDBClustersInput{}); err != nil {
    log.Println(err.Error())
}
```